src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -10635,3 +10677,123 @@ definitions:
         type: object
         additionalProperties:
           type: string
+  RulePaging:
+    type: object
+    required:
+      - list
+    properties:
+      list:
+        type: object
+        required:
+          - pagination
+          - entries
+        properties:
+          pagination:
+            $ref: '#/definitions/Pagination'
+          entries:
+            type: array
+            items:
+              $ref: '#/definitions/RuleEntry'
+  RuleEntry:
+    type: object
+    required:
+      - entry
+    properties:
+      entry:
+        $ref: '#/definitions/Rule'
+  Rule:
+    type: object
+    required:
+      - id
+      - name
+      - conditions
```

Comment:
Are conditions really required at the REST API level. I know that saving Rule at the model level will likely require a `no-condition` Condition object (at least that looks to be the case now in Share) but since we have a `trigger` field below, we may not want to add any conditions.

Reply:
Sure - I agree we can make it optional.